### ISSUE TITLE
tflint should maybe catch invalid AWS Route Table names

### ISSUE THREAD

**Issue by n8gard** [wata727/tflint#87]
If the AWS API requires strict formatting then maybe tflint could/should catch this?
`tflint` and `terraform plan` completed successfully but `terraform apply` produced:
```Error applying plan:

2 error(s) occurred:

* aws_route.nat-ngw-a: 1 error(s) occurred:

* aws_route.nat-ngw-a: Error creating route: InvalidRouteTableId.Malformed: Invalid id: "rtb-nat-gw-a"
	status code: 400, request id: 9eff0fe5-d8ec-468d-98a8-bd4b72ebe851
* aws_route.nat-ngw-b: 1 error(s) occurred:

* aws_route.nat-ngw-b: Error creating route: InvalidRouteTableId.Malformed: Invalid id: "rtb-nat-gw-b"
	status code: 400, request id: 8c0392c6-dd45-4f38-851f-7f316f84e589```

**Comment by wata727** [comment]
In such a case, I think that should be detect the issue when uses invalid `route_table_id` (it's not only about formatting) in template.
I'm planning to increase detectors that detect templates using such invalid references.

**Comment by wata727** [comment]
Hi @n8gard, TFLint v0.3.4 has been released. This release includes new detectors for `aws_route` resource. Please try it :)
